Source/Core/VideoCommon/ShaderGenCommon.h

```diff
@@ -214,12 +214,12 @@ class UidChecker
 };
 
 template<class T>
-inline void DefineOutputMember(T& object, API_TYPE api_type, const char* qualifier, const char* in_out, const char* type, const char* name, int var_index, const char* semantic = "", int semantic_index = -1)
+inline void DefineOutputMember(T& object, API_TYPE api_type, const char* qualifier, const char* type, const char* name, int var_index, const char* semantic = "", int semantic_index = -1)
 {
 	if (qualifier != nullptr)
-		object.Write("\t%s %s %s %s", qualifier, in_out, type, name);
+		object.Write("\t%s %s %s", qualifier, type, name);
 	else
```

Comment:
Hmm, so it was there before (from what I could tell) because of D3D and generating the structs. But those have had parameters added (from the other PR), so we could probably remove this now. Seems a bit cleaner.